### PR TITLE
Added anime to the list of phobias in phobia.dm, and traumas.dm.

### PR DESCRIPTION
**Pull request by SkippuSetsuna**
Added anime to phobia.dm, and SStrauma.dm and the items that are associated with it. Thanks to @XDTM for pointed out that I had to make changes in SStraumas in order for it to work.

### CODE REVIEW

**Comment by SgtxRootbeer** [comment]
Should be a default phobia

**Comment by CitrusGender** [comment]
nice

ememe

File: strings/phobia.json [hunk 1]
```diff
@@ -257,5 +257,17 @@
...
+		"anime",
+		"manga",
+		"baka",
+		"desu",
+		"sugoi",
+		"katana",
+		"weeb",
+		"otaku",
+		"catgirl",
+		"catgirls"
```
Comment (Mark9013100) [inline strings/phobia.json]: add uguu

**Comment by 81Denton** [comment]
Things I'd add
- katanas, schoolgirl uniforms, ninja outfits/equipment, sake/sashimi etc. to phobia_objs
- felinids to phobia_species

**Comment by 81Denton** [comment]
oh also dakimakura if those unironically got merged

**Comment by Kagus** [comment]
I'm not sure there's really a difference anymore between irony and unirony.

File: code/controllers/subsystem/traumas.dm [hunk 2]
```diff
@@ -135,9 +136,14 @@ SUBSYSTEM_DEF(traumas)
 						/obj/item/clothing/suit/chickensuit, /obj/item/clothing/head/chicken,
 						/obj/item/clothing/suit/toggle/owlwings, /obj/item/clothing/under/owl, /obj/item/clothing/mask/gas/owl_mask,
 						/obj/item/clothing/under/griffin, /obj/item/clothing/shoes/griffin, /obj/item/clothing/head/griffin,
-						/obj/item/clothing/head/helmet/space/freedom, /obj/item/clothing/suit/space/freedom))
-					   )
-
+						/obj/item/clothing/head/helmet/space/freedom, /obj/item/clothing/suit/space/freedom)),
+						
+					   "anime = typecahceof(list(/obj/item/clothing/under/schoolgirl, /obj/item/clothing/under/schoolgirl/red,
```
Comment (XDTM) [inline code/controllers/subsystem/traumas.dm]: missing a closing " after anime

File: code/controllers/subsystem/traumas.dm [hunk 3]
```diff
@@ -135,10 +136,15 @@ SUBSYSTEM_DEF(traumas)
 						/obj/item/clothing/suit/chickensuit, /obj/item/clothing/head/chicken,
 						/obj/item/clothing/suit/toggle/owlwings, /obj/item/clothing/under/owl, /obj/item/clothing/mask/gas/owl_mask,
 						/obj/item/clothing/under/griffin, /obj/item/clothing/shoes/griffin, /obj/item/clothing/head/griffin,
-						/obj/item/clothing/head/helmet/space/freedom, /obj/item/clothing/suit/space/freedom))
-					   )
-
-	phobia_turfs = list("space" = typecacheof(list(/turf/open/space, /turf/open/floor/holofloor/space, /turf/open/floor/fakespace)),
+						/obj/item/clothing/head/helmet/space/freedom, /obj/item/clothing/suit/space/freedom)),
+						
+					   "anime" = typecahceof(list(/obj/item/clothing/under/schoolgirl, /obj/item/clothing/under/schoolgirl/red,
```
Comment (Anonmare) [inline code/controllers/subsystem/traumas.dm]: You only need /obj/item/clothing/under/schoolgirl, the specific colours will be included with it

File: code/controllers/subsystem/traumas.dm [hunk 4]
```diff
@@ -135,10 +136,15 @@ SUBSYSTEM_DEF(traumas)
...
+					   /obj/item/clothing/under/schoolgirl/green, /obj/item/clothing/under/schoolgirl/orange, /obj/item/toy/katana, /obj/item/katana,
+					   /obj/item/nullrod/claymore/katana, /obj/item/energy_katana, /obj/item/reagent_containers/food/snacks/sashimi,
+					   /obj/item/reagent_containers/food/drinks/bottle/sake, /obj/item/throwing_star/ninja, /obj/item/throwing_star))
```
Comment (Anonmare) [inline code/controllers/subsystem/traumas.dm]: Ditto for throwing stars /obj/item/throwing_star/

File: code/controllers/subsystem/traumas.dm (hunk 3 above)
Comment (SkippuSetsuna) [reply code/controllers/subsystem/traumas.dm]: sweet thanks!

File: code/controllers/subsystem/traumas.dm [hunk 5]
```diff
@@ -135,10 +136,13 @@ SUBSYSTEM_DEF(traumas)
...
-					   )
-
-	phobia_turfs = list("space" = typecacheof(list(/turf/open/space, /turf/open/floor/holofloor/space, /turf/open/floor/fakespace)),
+						/obj/item/clothing/head/helmet/space/freedom, /obj/item/clothing/suit/space/freedom)),
+						
+					   "anime" = typecahceof(list(/obj/item/clothing/under/schoolgirl, /obj/item/katana, /obj/item/reagent_containers/food/snacks/sashimi,
+					   /obj/item/reagent_containers/food/drinks/bottle/sake, /obj/item/throwing_star))
+						)
+						
+	phobia_turfs = list("space" = typecahceof(list(/turf/open/space, /turf/open/floor/holofloor/space, /turf/open/floor/fakespace)),
```
Comment (Qustinnus) [inline code/controllers/subsystem/traumas.dm]: typecahceof(

File: code/controllers/subsystem/traumas.dm [hunk 6]
```diff
@@ -135,10 +136,13 @@ SUBSYSTEM_DEF(traumas)
 						/obj/item/clothing/suit/chickensuit, /obj/item/clothing/head/chicken,
 						/obj/item/clothing/suit/toggle/owlwings, /obj/item/clothing/under/owl, /obj/item/clothing/mask/gas/owl_mask,
 						/obj/item/clothing/under/griffin, /obj/item/clothing/shoes/griffin, /obj/item/clothing/head/griffin,
-						/obj/item/clothing/head/helmet/space/freedom, /obj/item/clothing/suit/space/freedom))
-					   )
-
-	phobia_turfs = list("space" = typecacheof(list(/turf/open/space, /turf/open/floor/holofloor/space, /turf/open/floor/fakespace)),
+						/obj/item/clothing/head/helmet/space/freedom, /obj/item/clothing/suit/space/freedom)),
+						
+					   "anime" = typecahceof(list(/obj/item/clothing/under/schoolgirl, /obj/item/katana, /obj/item/reagent_containers/food/snacks/sashimi,
```
Comment (Qustinnus) [inline code/controllers/subsystem/traumas.dm]: typecahceof(

**Comment by GuppyLaxx** [comment]
You forgot "Nya"

**Comment by Iamgoofball** [comment]
@81Denton hippie added them iirc

**Comment by XDTM** [comment]
I'm not particularly against this but remember that the more "specific" phobias there are the less effective it becomes, since they are rarely triggered.

By the way, felinids should probably be added as a trigger_species

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**